roles/openshift_master/tasks/main.yml

```diff
@@ -27,6 +27,9 @@
   action: "{{ ansible_pkg_mgr }} name={{ openshift.common.service_type }}-master{{ openshift_version | default('') | oo_image_tag_to_rpm_version(include_dash=True) }} state=present"
   when: not openshift.common.is_containerized | bool
 
+- name: Install bash completion for oc tools
+  action: "{{ ansible_pkg_mgr }} name=bash-completion state=present"
```

Comment:
@tbielawa That is currently the case, but the openshift_cli role is responsible for installing the client tools. I would also suggest to add a `when not openshift.common.is_containerized | bool` so that it doesn't attempt to install bash-completion for a containerized install.